doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/getting-started.ipynb

```diff
@@ -432,74 +332,39 @@
   },
   {
    "cell_type": "markdown",
-   "id": "c5d52948",
+   "id": "f8453b2a",
    "metadata": {},
    "source": [
-    "### Viewing the Ray Dashboard\n",
-    "\n",
-    "Assuming the port-forwarding process described above is still running, you may view the {ref}`ray-dashboard` by visiting `localhost:8265` in you browser.\n",
-    "\n",
-    "The dashboard port will not be used in the rest of this guide. You may stop the port-forwarding process if you wish.\n",
-    "\n",
-    "(kuberay-client)=\n",
-    "### Accessing the cluster using Ray Client\n",
-    "\n",
-    "[Ray Client](https://docs.ray.io/en/latest/cluster/ray-client.html) allows you to interact programatically with a remote Ray cluster using the core Ray APIs.\n",
-    "To try out Ray Client, first make sure your local Ray version and Python minor version match the versions used in your Ray cluster. The Ray cluster in our example is running Ray 2.0.0 and Python 3.7, so that's what we'll need locally. If you have a different local Python version and would like to avoid changing it, you can modify the images specified in the yaml file `ray-cluster.autoscaler.yaml`. For example, use `rayproject/ray:2.0.0-py38` for Python 3.8.\n",
-    "\n",
-    "After confirming the Ray and Python versions match up, the next step is to port-forward the Ray Client server port (10001 by default).\n",
-    "If you are following along in a Jupyter notebook, execute the following command in a separate shell."
+    "For a more detailed guide on using Ray Jobs to run applications on a Ray Cluster, check out the [quickstart guide](jobs-quickstart-under-construction)"
```

Comment:
Since the user has already set up a connection, we might as well provide a snippet here that submits a simple job.

Reply:
Oh sry, that's still there.